app/views/layouts/application.html.haml

```diff
@@ -14,20 +14,20 @@
     - # container for right/main panel
     .md:col-span-3.bg-blue-100.flex-1.flex.flex-col
 
-      - # notice box
-      - if flash[:notice]
-        #notice.mx-auto.my-4{:class => "w-5/6 md:w-2/3"}
-          .bg-teal-100.border-t-4.border-teal-500.rounded-b.text-teal-900.px-4.py-3.shadow-md{:role => "alert"}
-            %button#close-notice.bg-transparent.hover:bg-red-200.float-right.p-1.rounded
-              %svg.fill-current.h-4.w-4.text-red-600{:role => "button", :viewbox => "2 2 16 16", :xmlns => "http://www.w3.org/2000/svg"}
-                %title Close
-                %path{:d => "M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"}
-            .flex
-              .py-1
-                %svg.fill-current.h-5.w-5.text-teal-500.mr-4{:viewbox => "0 0 20 20", :xmlns => "http://www.w3.org/2000/svg"}
-                  %path{:d => "M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z"}
-              %div
-                %p.text-md= flash[:notice]
+        - # notice box
```

Comment:
This indentation part breaks the page on my machine:
![konyha_break](https://user-images.githubusercontent.com/25415958/95016108-336f9380-0651-11eb-932b-11a48cac6e32.png)

Reply:
I made a mistake while merging master. Fixing it in next commit.